features/steps/store_resource.py

```diff
@@ -215,16 +105,16 @@ def get_resources(self):
 
 # Handlers
 
-def receive_package(event: PackageReceived, uow: FakeUnitOfWork) -> None:
+def receive_package(event: PackageSubmitted, uow: UnitOfWork) -> None:
```

Comment:
@jayamala17 helped identify this bug (incorrect event type), which I must have introduced at some point.